css-ecommerce-page/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row text-align">
+      <div class="column-full">
+        <h2>New Arrivals</h2>
+      </div>
+    </div>
+
+    <div class="row justify-content">
+      <div class="column-full">
+        <div class="row">
+          <div class="column-imgs">
+            <img src="images/dress1.jpeg" alt="black-dress">
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-full margin">
+            <span>Free People</span>
+            <span class="light-gray-font">| Size 6</span>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-full margin">
```

Comment:
I would make these elements into a `p` rather than a `div`, since this is only holding text content.